shanoir-ng-datasets/src/main/java/org/shanoir/ng/configuration/amqp/RabbitMQDatasetsService.java

```diff
@@ -313,7 +313,7 @@ public void createDatasetAcquisition(final String studyStr) {
 
 	/**
          * Receives a shanoirEvent as a json object, concerning a subject deletion
-         * @param eventAsString the task as a json string.
+         * @param subjectIdAsString a string of the subject's id
          */
 	@RabbitListener(bindings = @QueueBinding(

```

Comment:
Please remove this and update it to be simpler (and not event depending) :
	@RabbitListener(queues = RabbitMQConfiguration.DELETE_SUBJECT_QUEUE, containerFactory = "singleConsumerFactory")